python/nano/tutorial/notebook/inference/pytorch/accelerate_pytorch_inference_jit_ipex.ipynb

```diff
@@ -15,27 +15,33 @@
       ]
     },
     {
+      "attachments": {},
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "> 📝 **Note**\n",
-        ">\n",
-        "> * `jit`: You can use `InferenceOptimizer.trace(..., accelerator=\"jit\")` API to enable the jit acceleration for PyTorch inference.\n",
-        "> * `ipex`: You can use`InferenceOptimizer.trace(...,use_ipex=True)` API to enable the ipex acceleration for PyTorch inference. It only takes a few lines.\n",
-        "> * `jit + ipex`: It is recommended to use JIT and IPEX together. You can user `InferenceOptimizer.trace(..., acclerator=\"jit\", use_ipex=True`) to enable both for PyTorch inference."
+        "* JIT: You can use `InferenceOptimizer.trace(..., accelerator=\"jit\")` API to enable the TorchScript acceleration for PyTorch inference.\n",
+        "* IPEX: You can use `InferenceOptimizer.trace(...,use_ipex=True)` API to enable the IPEX (Intel® Extension for PyTorch*) acceleration for PyTorch inference.\n",
+        "* JIT + IPEX: It is recommended to use JIT and IPEX together. You can user `InferenceOptimizer.trace(..., acclerator=\"jit\", use_ipex=True`) to enable both for PyTorch inference.\n",
+        "\n",
+        "All of the above accelerations only take a few lines to apply."
       ]
     },
     {
+      "attachments": {},
       "cell_type": "markdown",
-      "metadata": {},
+      "metadata": {
+        "nbsphinx": "hidden"
+      },
       "source": [
-        "To apply JIT/IPEX acceleration, the following dependencies need to be installed first："
+        "To apply JIT/IPEX acceleration, you should install BigDL-Nano for PyTorch first："
```

Comment:
`BigDL-Nano for PyTorch` -> It feels strange to read here

Reply:
As discussed, I will keep it as it is for now.

May change the words for the installation related description later if needed. Suggested changes:
- For [pytorch]: install BigDL alongside with PyTorch
- For [pytorch,inference]: install BigDL alongside with PyTorch, and related inference backends